generators/server/files.js

```diff
@@ -343,7 +343,7 @@ const serverFiles = {
         {
             path: DOCKER_DIR,
             condition: generator => generator.buildTool === 'maven',
-            templates: ['Dockerfile.jvm', 'Dockerfile.native']
+            templates: ['Dockerfile.jvm', 'Dockerfile.native', 'Dockerfile.fast-jar']
```

Comment:
When the Maven/Gradle condition will be removed, I suggest to add those files to the expect-files.js to improve our test coverage.